mantidproject/mantid
# Handle transmission for loq better

Fixes #14068 

This issue addresses the following problems related to transmission settings in the GUI:
- M4 is availble when selecting LOQ instrument
- the z-shift is available when selecting M3
- all check boxes can be unselected
- when loading from LOQ nothing is checked
# For testing

The test files are located here: \olympic\Babylon5\Scratch\Anton\Testing\14068
There is a sub folder SANSTest and a subfolder LOQTest.

**Check Checkbox logic**
1. Set SANSTest in the Mantid directories
2. Set SANS2DTubes as the instrument
3. Load the user file user_file_M3_old.txt (in the folder)
4. Navigate to the Reduction Settings tab ( we focus on the Transmssion Settings section)
   - Confirm that M3 is checked (this is set in the User file)
   - Confirm that the shift field is not editable
5. Try to uncheck
   - Confirm that one cannot uncheck it
6. Check M4
   - Confirm that the shift field becomes editable
7. Check the Radius check box and the ROI files checkbox
   - Confirm that M4 is unchecked
8. Uncheck the Radius and ROI files checkboxes
   - Confirm that M3 is now checked.  (When nothing is checked, we default back to M3)
9. Change the instrument from SANS2DTubes to LOQ on the Run Numbers tab
   - Confirm that M4 is disabled
10. Change the instrument from LOQ to SANS2DTubes on the Run Numbers tab

**Check when loading LOQ user file**
1. Set LOQTest in the Mantid directories
2. Set LOQ as the instrument
3. Load the user file USER_LOQ_151D_Xpress_6mm.txt (in the folder)
   - Confirm that M3 is checked
## For Release notes:

See [here](http://www.mantidproject.org/index.php?title=ReleaseNotes_3_6_SANS&diff=25414&oldid=25332)
